src/status_im2/common/bottom_sheet/view.cljs

```diff
@@ -26,11 +26,11 @@
             options)
    callback))
 
-(def content-height (reagent/atom nil))
+(def ^:private content-height (reagent/atom nil))
 (def show-bottom-sheet? (reagent/atom nil))
-(def keyboard-was-shown? (reagent/atom false))
-(def expanded? (reagent/atom false))
-(def gesture-running? (reagent/atom false))
+(def ^:private keyboard-was-shown? (reagent/atom false))
```

Comment:
Great, thanks for helping isolate global state. It's a step in the right direction.